cranelift-codegen/meta/src/shared/formats.rs

```diff
@@ -169,6 +169,11 @@ pub fn define(immediates: &OperandKinds, entities: &OperandKinds) -> FormatRegis
             .imm(("src", stack_slot))
             .imm(("dst", regunit)),
     );
+    registry.insert(
+        Builder::new("FillNop")
```

Comment:
Do we actually need this new format? It seems to me that it is unused, since the encodings use the `f_unary` format instead. (If that's true, then we'd need the meta crate to warn about unused formats, because they'll generate dead code.)

Reply:
Ah, well spotted.  It seems to work fine if I remove it.